setup.cfg

```diff
@@ -6,3 +6,7 @@ filterwarnings=
     ignore:.*formatargspec.*:DeprecationWarning
     ignore:.*isAlive.*:PendingDeprecationWarning
 
+[flake8]
+extend-ignore = E501,E203
+exclude = venv,tests
```

Comment:
How come we're ignoring tests here? Also, so we need to add venv here and above? Since we should be using pipenv to create the virtualenv?

Reply:
The reason we're ignoring is because we would then have to refactor a large amount of Python files and not ignoring them leads to lots of lint errors. From my understanding, I had to ignore venv here as well because of the way my project is set up otherwise I still get the bunch of lint errors.

Reply:
I don't think we can exclude tests though as this will ignore the python test files that should be linted in the test directory

Reply:
But did we lint tests before? If not we should create another PR to fix linting issues in tests?

Reply:
We use this config for flake8 so maybe we should paste it into the new config:
```
[flake8]
# Ignore node_modules and cloned repos when not in a virtual environment
exclude = node_modules/*,tests/*,src/*
max-line-length = 160
ignore = C815,C816,W503,E203
```

Reply:
Think @petechd is correct, the config should be the same as above 👍

Reply:
Why is it in setup.cfg?. It is already configured in `tox.ini`, we only want 1 source of truth. Either is fine by me as long as there is only one. Is there some specific reason both are needed?

Reply:
I've made the changes, let me know if they are suitable